lib/dictBuilder/cover.c

```diff
@@ -529,7 +529,7 @@ static void COVER_ctx_destroy(COVER_ctx_t *ctx) {
  * Returns 1 on success or zero on error.
  * The context must be destroyed with `COVER_ctx_destroy()`.
  */
-static int COVER_ctx_init(COVER_ctx_t *ctx, const void *samplesBuffer,
+size_t COVER_ctx_init(COVER_ctx_t *ctx, const void *samplesBuffer,
```

Comment:
The CI error is a bit cryptic, but it is basically just telling you that you need to mark this function `static`.

`static` means that this function is only used in this translation unit, so don't export it. All functions not marked static are implicitly exported.

As a general rule, unless a function is present in a header file, it should be marked static.